Sources/NetworkProtection/Notifications/NetworkProtectionNotification.swift

```diff
@@ -82,12 +84,16 @@ public enum NetworkProtectionNotificationName: String {
     case issuesStarted = "com.duckduckgo.network-protection.NetworkProtectionNotification.issuesStarted"
     case issuesResolved = "com.duckduckgo.network-protection.NetworkProtectionNotification.issuesResolved"
 
+    // User Notification Events
+    case showIssuesStartedNotification = "com.duckduckgo.network-protection.NetworkProtectionNotification.showIssuesStartedNotification"
+    case showIssuesResolvedNotification = "com.duckduckgo.network-protection.NetworkProtectionNotification.showIssuesResolvedNotification"
+    case showIssuesNotResolvedNotification = "com.duckduckgo.network-protection.NetworkProtectionNotification.showIssuesNotResolvedNotification"
+    case showVPNSupersededNotification = "com.duckduckgo.network-protection.NetworkProtectionNotification.showVPNSupersededNotification"
+    case showTestNotification = "com.duckduckgo.network-protection.NetworkProtectionNotification.showTestNotification"
```

Comment:
New distributed notifications to ask the notifications App to present user notifications.